PhpStorm Protocol (Win)/run_editor.js

```diff
@@ -6,7 +6,12 @@ var settings = {
 	folder_name: 'PhpStorm 9.0',
 
 	// Set to window title (only text after dash sign), that you see, when switching to running PhpStorm instance
-	window_title: 'PhpStorm 9.0'
+	window_title: 'PhpStorm 9.0',
+	
+	// In case your file is mapped via a network share and paths do not match.
+	// eg. /var/www will be replaced with Y:/
+	projects_basepath: '',

```

Comment:
This and below setting name doesn't match to used settings in 31 and 32 lines. I guess you've pushed your changes too early.